gridstatus/tests/test_ercot.py

```diff
@@ -817,6 +840,9 @@ def test_get_rtm_price_corrections(self):
         assert df.shape[0] >= 0
         assert df.columns.tolist() == cols
 
+    # TODO: this url has no DocumentList
+    # https://www.ercot.com/misapp/servlets/IceDocListJsonWS?reportTypeId=13044
```

Comment:
The document list at this URL is empty:

```json
{
  "ListDocsByRptTypeRes": {
  "DocumentList": []
  }
}
```

Reply:
i think this because this endpoint only stores last 180 days of price corrections, and there hasn't been one since then.